app/views/users/partials/_show_partial.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>Friend requests received from:</p><br>
+<% current_user.friend_request_received.each do |frr| %>
+  <ul>
+    <li>
+      <strong><%= frr.name %></strong>
+      <button><%= link_to 'Accept', "friendship/accept/#{frr.id}", method: :post %></button>
+      <button><%= link_to 'Reject', "friendship/reject/#{frr.id}", method: :post %></button>
+    </li>
```

Comment:
I can't find this in the app. I can send an invitation but on the other side nothing happens.
Please add it next to the user's name in the "All Users" menu and in the User's profile. [Check requirement. ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/pending_request.png)

**Optional**: you can create a new menu called Notifications and add the friendship requests there. If you feel like it, you can even add a bootstrap batch to show the number of notifications.